src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -804,7 +804,6 @@ TTL7414 = 7414: hex inverter (schmitt-trigger)
 TTL74161 = 74161: 4-bit sync counter with async clear
 TTL74163 = 74163: 4-bit sync counter with sync clear
 TTL74164 = 74164: 8-bit serial-to-parallel shift register
-TTL74165 = 74165: 8-bit parallel-to-serial shift register with asynchronous load
 TTL74165 = 74165: 8-bit parallel-to-serial shift register
```

Comment:
As far as I can see from [the datasheet](https://www.ti.com/lit/ds/symlink/sn54ls165a-sp.pdf?ts=1748424724545), the load is indeed asynchronous.